sirepo/package_data/static/js/cloudmc.js

```diff
@@ -2705,13 +2706,14 @@ SIREPO.app.directive('jRangeSlider', function(appState, panelState) {
             model: '=',
             modelName: '<',
             onChange: '&',
+            sliderIndex: '=',
```

Comment:
sliderIndex is not the best thing to use here; there are likely other conditions that may require hiding the labels. Something like hideMinMax is more flexible